src/System.Windows.Forms/src/System/Windows/Forms/Form.cs

```diff
@@ -1516,6 +1516,14 @@ public Form[] MdiChildren
             }
         }
 
+        /// <summary>
+        /// true - anchoring minimized MDI children to the BottomLeft of the parent form (WinForms default). false - anchoring minimized MDI children to the TopLeft of the parent form (windows default).
+        /// </summary>
```

Comment:
Please have a look how all other properties are described. The summary wording must start with something like "Gets or sets...", and it mustn't contain value's information.


```suggestion
        /// <summary>
        /// Gets or sets the anchoring for minimized MDI children.
        /// </summary>
        /// <value><see langword="true" /> to anchor minimized MDI children to the bottom left of the parent form; <see langword="false" /> to anchor to the top left of the parent form (Windows default).</value>
```

Reply:
Got it, thanks.